scripts/aem.js

```diff
@@ -335,11 +353,12 @@ function transformToCustomElement(brick) {
 }
 
 function getBrickResources() {
-  const components = new Set();
-  const templates = new Set();
+  const components = new Set(['aem-root', 'aem-header', 'aem-footer']);
+  const templates = new Set(['aem-root', 'aem-header', 'aem-footer']);
 
-  document
-    .querySelectorAll('header, footer, div[class]:not(.fragment):not(.section)')
+  // Load Bricks
+  document.body
+    .querySelectorAll('div[class]:not(.fragment)')
```

Comment:
Iff we want to allow a mix of bricks and blocks, for existing projects that want to gradually move to bricks, I suppose we'd need a function (supplied by `config.js` probably) here to decide what's a brick and what's a block.